bindings/c/test/apitester/TesterUtil.cpp

```diff
@@ -106,4 +106,24 @@ KeyRangeArray copyKeyRangeArray(fdb::future_var::KeyRangeRefArray::Type array) {
 	return out;
 };
 
+GranuleSummaryArray copyGranuleSummaryArray(fdb::future_var::GranuleSummaryRefArray::Type array) {
+	auto& [in_summaries, in_count] = array;
+
+	GranuleSummaryArray out;
+
+	for (int i = 0; i < in_count; ++i) {
+		fdb::native::FDBGranuleSummary nativeSummary = *in_summaries++;
+		fdb::GranuleSummary summary;
+		summary.keyRange.beginKey =
```

Comment:
It would be probably nicer to implement this copying logic as a GranuleSummary constructor, possibly also reusing it in other places